docs/src/dev-guide/building-package.md

```diff
@@ -22,11 +23,11 @@ Initialize the project
 tools/scripts/deps-download/init.sh
 ```
 
-Install CLP core's dependencies
-
-```shell
-components/core/tools/ubuntu-focal/install-all.sh
-```
+Install CLP core's dependencies based on your OS:
+* [CentOS Stream 9](https://docs.yscope.com/clp/main/dev-guide/components-core/centos7.4-deps-install.html)
+* [macOS 12](https://docs.yscope.com/clp/main/dev-guide/components-core/macos12-deps-install.html)
+* [Ubuntu 20.04](https://docs.yscope.com/clp/main/dev-guide/components-core/ubuntu-focal-deps-install.html)
+* [Ubuntu 22.04](https://docs.yscope.com/clp/main/dev-guide/components-core/ubuntu-jammy-deps-install.html)
```

Comment:
Unfortunately, trying to build the package on OSes other than Ubuntu Focal won't always work.

As the docs above say:

> At runtime, the CLP package uses an Ubuntu Focal container, so we need to build in a matching environment.

If we build the package using CentOS, it's unlikely that the core binaries can run in the Ubuntu Focal container we use at runtime. Iirc, @junhaoliao had added support for also building on Ubuntu Jammy, but since the docs are already a bit confusing, I think that's best left as a hidden feature for now.

At some point, we will add support for building the package on any OS, but it's just not a priority right now.

Note that the [core](https://docs.yscope.com/clp/main/dev-guide/components-core/index.html) binaries are different from the [package](https://docs.yscope.com/clp/main/dev-guide/building-package.html). The package is a distributed system with many components whereas the core binaries are the component that does the actual compression, decompression, and search.

Reply:
tl;dr, I think we need to undo this change and revert to your previous fix of the Ubuntu Focal script path.

Reply:
Makes sense. Change reverted.